Cleaned up Docker compose file

jlaacke opened the pull request:
- Removed voorwiel from docker compose file since it is always built separately
- Always restart cykel related docker containers to use them like daemon processes
- Removed ALLOWED_HOSTS, since they are configured via .env file

robbi5 commented:
Thanks for the PR :) 
Just one question: do you have a special reason to add the `api/stations/location` endpoint for gps coordinates of stations, or could you use the ones already provided by the [station_information.json from GBFS](https://github.com/NABSA/gbfs/blob/v2.2/gbfs.md#station_informationjson) (https://github.com/stadtulm/cykel/blob/master/gbfs/serializers.py#L62)?

jlaacke commented:
@robbi5 You are absolutely right, we don't need an additonal API for that. So I removed that and changed the title of the PR. ;) The voorwiel PR is ready to go: https://github.com/stadtulm/voorwiel/pull/28
